tests/lib/rules/no-control-regex.js

```diff
@@ -20,16 +20,17 @@ var ruleTester = new RuleTester();
 
 ruleTester.run("no-control-regex", rule, {
     valid: [
-        "var regex = /x1f/",
+        "var regex =" + /x1f/,
```

Comment:
Why is this written as a string concat? This is a constant value, just write the string out.